scripts/list_updatable_packages

```diff
@@ -17,14 +17,26 @@ SESSION = requests.Session()
 Package = namedtuple('Package', ['package_name', 'version', 'gem_name', 'directory'])
 
 
-def find_packages() -> Generator[Package, None, None]:
+def find_package_directories() -> Generator[tuple, None, None]:
```

Comment:
I think `tuple` here is a Python 3.8 feature. At least a Python that's not available in `ubuntu-latest`. IIRC you'll need to import `Tuple` from typing.

Reply:
:disappointed: 

This slow addition of typing makes me so sad sometimes.